lib/internal/per_context/primordials.js

```diff
@@ -124,5 +124,14 @@ primordials.SafePromise = makeSafe(
   copyPrototype(original.prototype, primordials, `${name}Prototype`);
 });
 
+// Helper caried over from js2c_macros: exits process and prints debug message
+// iff node_debug_lib flag is set.
+primordials.CHECK = function(x) {
```

Comment:
Is `primordials` the right place for this? Might it be better moved to `lib/internal/vm/module.js` if that's the only place using it? I guess it comes down to: Is this a use we want to discourage ro accommodate?

Reply:
I'd be open to moving the helper into `internal/vm/module.js`, I think it would be prudent to leave the functionality in this module for the time being.